WalletWasabi/WabiSabi/Client/CoinJoin/Client/CoinJoinClient.cs

```diff
@@ -126,6 +126,17 @@ private async Task<RoundState> WaitForBlameRoundAsync(uint256 blameRoundId, Canc
 			throw new InvalidOperationException($"Blame Round ({roundState.Id}): Abandoning: the round is not economic.");
 		}
 
+		if (roundState.CoinjoinState.Parameters.CoordinationFeeRate.Rate * 100 > CoinJoinConfiguration.MaxCoordinationFeeRate)
+		{
+			Logger.LogWarning("The coordinator is malicious and tried to trick the client into paying a higher coordination fee rate in the blame round.");
+			throw new InvalidOperationException($"Blame Round ({roundState.Id}): Abandoning: the coordination fee rate is too high.");
+		}
+
+		if (roundState.CoinjoinState.Parameters.MiningFeeRate.SatoshiPerByte > CoinJoinConfiguration.MaxCoinJoinMiningFeeRate)
```

Comment:
WTF is a `MaxCoinJoinMiningFeeRate`!!?? It is a coordination fee or a mining fee? 
Could you please rename it in a way that make some sense? Also please make rates rates and not percentages.

Reply:
> WTF is a `MaxCoinJoinMiningFeeRate`!!?? It is a coordination fee or a mining fee? Could you please rename it in a way that make some sense? Also please make rates rates and not percentages.

I think this name is straightforward to me.

Reply:
Please suggest a name because this one is perfectly correct to me. 
I will rename the other internally `MaxCoordinationFeePercent`

Reply:
You both are right, it didn't make sense to me but I understand now. Let it as it is.